projects/packages/videopress/src/client/components/video-upload-area/style.module.scss

```diff
@@ -0,0 +1,64 @@
+.wrapper {
+	max-width: 552px;
```

Comment:
Don't think we should put a `max-width`, this should be the parent responsibility IMO.